app/views/users/show.html.erb

```diff
@@ -1 +1,35 @@
-<h1>This page shows single user by specific id</h1>
\ No newline at end of file
+<main>
+    <div class="single-user-container">
+        <div class="single-user">
+            <div class="user-image">
+                <img src="<%= @user.photo%>" alt="profile" class="user-profile">
+            </div>
+            <div class="user-information">
+                <h2 class="user-name"><%= @user.name%></h2>
+                <span class="post-counter"><p class="count">Number of posts: <%= @user.posts.count %></p></span>
+            </div>
+        </div>
+        <div class="user-bio">
+            <h4 class="bio">Bio</h4>
+            <p class="bio-paragraph">
+                <%= @user.bio %>
+            </p>
+        </div>
+        <div class="user-all-posts">
+            <% @user.posts.each_with_index do |post,index|%>
```

Comment:
- [ ] As per the requirement, only the 3 most recent posts of the user must be rendered here. Kindly implement this.
![image](https://github.com/Javed-Shahbaz/Blog-App/assets/91486791/fa1c34a6-b41e-4dfe-90ad-3c09813d45eb)
(your model)